### ISSUE TITLE
Unhandled exception: type 'Null' is not a subtype of type 'String'

### ISSUE THREAD

**Issue by 1440000bytes** [vishalxl/nostr_console#75]
```
Waiting for user posts to come in........done
Waiting for feed to come in..............done
Unhandled exception:
type 'Null' is not a subtype of type 'String'
#0      Store.handleChannelEvents (package:nostr_console/tree_ds.dart:815)
#1      new Store.fromEvents.<anonymous closure> (package:nostr_console/tree_ds.dart:1263)
#2      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:617)
#3      new Store.fromEvents (package:nostr_console/tree_ds.dart:1254)
#4      getTree (package:nostr_console/tree_ds.dart:2723)
#5      main.<anonymous closure>.<anonymous closure> (file:///work/bin/nostr_console.dart:426)
#6      new Future.delayed.<anonymous closure> (dart:async/future.dart:424)
#7      Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18)
#8      _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398)
#9      _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429)
#10     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192)
```

**Comment by vishalxl** [comment]
This is same as #70

**Comment by vishalxl** [comment]
Release with [fix is 0.3.5](https://github.com/vishalxl/nostr_console/releases/tag/v0.3.5-beta)
